prime-router/src/test/kotlin/SimpleReportTests.kt

```diff
@@ -177,129 +180,120 @@ class SimpleReportTests {
     }
 
-    @Test
-    @Ignore
-    fun `test fake FL data`() {
-        val schemaName = "fl/fl-covid-19"
-        val fakeReportFile = createFakeFile(schemaName, 100)
-        // Run the data thru its own schema and back out again
-        val fakeReportFile2 = readAndWrite(fakeReportFile.absolutePath, schemaName)
-        compareTestResultsToExpectedResults(fakeReportFile, fakeReportFile2, recordId = "Medical Record Number")
-    }
-
     @Test
     fun `test internal read and write`() {
         val schemaName = "az/pima-az-covid-19"
         val fakeReportFile = createFakeFile(schemaName, 100, useInternal = true)
         // Run the data thru its own schema and back out again
         val fakeReportFile2 = readAndWriteInternal(fakeReportFile.absolutePath, schemaName)
-        assertTrue(FileUtils.contentEquals(fakeReportFile, fakeReportFile2))
+        assertThat(FileUtils.contentEquals(fakeReportFile, fakeReportFile2)).isTrue()
     }
 
-    private fun compareTestResultsToExpectedResults(
-        testFile: File,
-        expectedResultsFile: File,
-        compareKeys: Boolean = true,
-        compareLines: Boolean = true,
-        recordId: String = "Patient_ID"
-    ) {
-        assertTrue(testFile.exists())
-        assertTrue(expectedResultsFile.exists())
-        // A bit of a hack:  diff the two files.
-        val testFileLines = testFile.readLines()
-        val expectedResultsLines = expectedResultsFile.readLines()
-
-        val testLines = convertFileToMap(testFileLines, recordId = recordId)
-        val expectedLines = convertFileToMap(expectedResultsLines, recordId = recordId)
-        val headerRow = expectedResultsLines[0].split(",")
-
-        if (compareKeys) {
-            // let's first compare the keys
-            compareKeysOfMaps(expectedLines, testLines)
-        }
-
-        if (compareLines) {
-            // let's compare our lines now
-            compareLinesOfMaps(expectedLines, testLines, headerRow)
-        }
-    }
+    companion object {
+        private fun convertFileToMap(
+            lines: List<String>,
+            recordId: String = "Patient_ID",
+            skipHeader: Boolean = true,
+            delimiter: String = ","
+        ): Map<String, Any?> {
+            val expectedLines = mutableMapOf<String, Any?>()
+            val headerLine = lines[0]
+            var recordIdIndex = 0
+            var skippedHeader = false
+
+            for (expectedResultsLine in lines) {
+                // if a header is passed in and we need to skip it, then check our control values
+                if (skipHeader && !skippedHeader) {
+                    // while we're in the header, find our record id index, which will be our map key
+                    val headerValues = headerLine.split(",")
+                    recordIdIndex = headerValues.indexOf(recordId)
+                    // reset our control variable
+                    skippedHeader = true
+                    continue
+                }
```

Comment:
I moved all of these functions in SimpleReportTests into the companion object because they are static and should live there instead.